kie-archetypes/kie-service-spring-boot-archetype/src/main/resources/META-INF/archetype-post-generate.groovy

```diff
@@ -0,0 +1,357 @@
+    indexFile.newWriter().withWriter { w ->
+        w << indexContent
+    }
+} else if(appType == "planner") {
+    println "Updating application properties...";
+    def appPropertiesContent = appPropertiesFile.getText('UTF-8');
+    def devAppPropertiesContent = devAppPropertiesFile.getText('UTF-8');
+
+    println "- adding server capabilities";
+    appPropertiesContent = appPropertiesContent.replace(kieServerCapabilitiesMarker, serverCapabilitiesPlanner);
+    devAppPropertiesContent = devAppPropertiesContent.replace(kieServerCapabilitiesMarker, serverCapabilitiesPlanner);
+
+    println "- removing jbpm configuration";
+    appPropertiesContent = appPropertiesContent.replace(jbpmConfigMarker, '');
+    devAppPropertiesContent = devAppPropertiesContent.replace(jbpmConfigMarker, '');
+
+    println "- removing database configuration";
+    appPropertiesContent = appPropertiesContent.replace(databaseConfigMarker, '');
+    devAppPropertiesContent = devAppPropertiesContent.replace(databaseConfigMarker, '');
+
+    appPropertiesFile.newWriter().withWriter { w ->
+        w << appPropertiesContent
+    }
+
+    devAppPropertiesFile.newWriter().withWriter { w ->
+        w << devAppPropertiesContent
+    }
+
+    println "Updating pom...";
+    def pomContent = pomFile.getText('UTF-8');
+
+    println "- adding spring boot starter dependency";
+    pomContent = pomContent.replace(springBootStarterMarker, PlannerSpringBootStarterDepends);
+
+    println "- remove db dependency";
+    pomContent = pomContent.replace(databaseDependsMarker, '');
+
+    pomFile.newWriter().withWriter { w ->
+        w << pomContent
+    }
+
+    println "Updating index.html...";
+    def indexContent = indexFile.getText('UTF-8');
+    indexContent = indexContent.replace(indexCSSMarkerBA, 'alert alert-danger');
+    indexContent = indexContent.replace(indexCSSMarkerDM, 'alert alert-success');
+    indexContent = indexContent.replace(indexCSSMarkerBO, 'alert alert-success');
+    indexContent = indexContent.replace(indexIconMarkerBA, 'fa fa-times-circle-o');
+    indexContent = indexContent.replace(indexIconMarkerDM, 'fa fa-check-circle-o');
+    indexContent = indexContent.replace(indexIconMarkerBO, 'fa fa-check-circle-o');
+    indexFile.newWriter().withWriter { w ->
+        w << indexContent
+    }
+} else {
+    println "[ERROR: Invalid app type specified - unable to finish needed configurations!]";
+}
+
+println "Updating launch scripts...";
+def launchFileContent = launchFile.getText('UTF-8');
+def devLaunchFileContent = devLaunchFile.getText('UTF-8');
+
+launchFileContent = launchFileContent.replaceAll(myServiceNameMarker, myAppArtifactId);
+launchFileContent = launchFileContent.replaceAll(myServiceVersionMarker, myAppVersion);
+devLaunchFileContent = devLaunchFileContent.replaceAll(myServiceNameMarker, myAppArtifactId);
+devLaunchFileContent = devLaunchFileContent.replaceAll(myServiceVersionMarker, myAppVersion);
```

Comment:
I believe this replace will generate incorrect setup
```
projects=( "test-service-model" "test-service-kjar" "test-service")
```
while the application name I created was test and thus the projects generated where like this
- test-model
- test-kjar
- test-service

so the launch script builds only test-service and skips the test-model and test-kjar